source/app/components/DocumentActionBar/DocumentActionBar.js

```diff
@@ -0,0 +1,60 @@
+import { Flex, Box, Button, HStack, useMediaQuery } from '@chakra-ui/react';
+import { useDispatch } from 'react-redux';
+import ExportPreview from './ExportPreview/ExportPreview';
+import { clearRedactions } from '../../store/entities/documents/actions';
+import DocumentSearchBar from '../DocumentSearchBar/DocumentSearchBar';
+import OrangeButton from '../Button/Button';
+import { ExclusionListMenu } from './ExclusionListMenu/ExclusionListMenu';
+import { getAreRedactionsOnDocument } from '../../utils/document';
+
+const DocumentActionBar = ({ document, isComplianceTrack, redactMatches }) => {
+  const dispatch = useDispatch();
+  const { documentId } = document;
+  const [showRedactionLegend] = useMediaQuery('(min-width: 1550px)');
+
+  return (
+    <Flex py={3} px='2.96875vw' justifyContent='space-between' alignItems='center'>
+      {isComplianceTrack && showRedactionLegend && (
+        <Box
+          aria-hidden
+          pointerEvents='none'
+          boxShadow='0 0 0 2px #000'
+          bg='rgba(128, 128, 128, 0.2)'
```

Comment:
For colors, should we stick to one model of color picking between RGBA and HEX?
For example, we could use RGBA all across or (personal preference) HEX + Opacity